cluster/src/main/java/com/hartwig/pipeline/execution/vm/InstanceLifecycleManager.java

```diff
@@ -128,21 +131,23 @@ private Operation executeSynchronously(ComputeRequest<Operation> request, String
                 Thread.currentThread().interrupt();
             }
         }
-        return executeWithRetries(() -> compute.zoneOperations().get(projectName, zoneName, asyncOp.getName()).execute());
+        return executeWithRetries(() -> compute.zoneOperations().get(projectName, zoneName, asyncOp.getName()).execute(),
+                IOException.class);
     }
 
-    private Operation executeWithRetries(final CheckedSupplier<Operation> operationCheckedSupplier) {
-        return Failsafe.with(new RetryPolicy<>().handle(IOException.class).withDelay(Duration.ofSeconds(pollInterval)).withMaxRetries(5))
+    private <T> T executeWithRetries(final CheckedSupplier<T> operationCheckedSupplier, final Class<? extends Exception> exception) {
+        return Failsafe.with(new RetryPolicy<>().handle(exception)
+                .withBackoff(pollInterval, FIVE_MINUTES, ChronoUnit.SECONDS, new Random().nextInt(pollInterval)))
```

Comment:
I wonder in retrospect whether the three-argument version of `withDelay` might be more appropriate (provide a min and max delay and it will choose a random time between them). I think it's more readable too (and you could do without the constant - just use `5` and specify minutes instead of seconds).